CMakePresets.json

```diff
@@ -1,41 +1,45 @@
 {
-    "version": 3,
-    "configurePresets": [
-        {
-            "name": "scap-drivers",
-            "displayName": "Build scap drivers and their tests",
-            "description": "Build all scap drivers (modern eBPF, legacy eBPF, kmod) and their tests",
-            "generator": "Unix Makefiles",
-            "binaryDir": "${sourceDir}/build-scap-drivers",
-            "cacheVariables": {
-                "CMAKE_BUILD_TYPE": "Release",
-                "BUILD_BPF": "ON",
-                "BUILD_DRIVER": "ON",
-                "USE_BUNDLED_DEPS": "ON",
-                "ENABLE_DRIVERS_TESTS": "ON",
-                "MODERN_BPF_DEBUG_MODE": "ON",
-                "BUILD_LIBSCAP_MODERN_BPF": "ON",
-                "BUILD_LIBSCAP_GVISOR": "OFF",
-                "CREATE_TEST_TARGETS": "ON",
-                "ENABLE_LIBSCAP_TESTS": "ON",
-                "SCAP_FILES_SUITE_ENABLE": "OFF"
-            }
-        },
-        {
-            "name": "sinsp-minimal",
-            "displayName": "Build sinsp in minimal build",
-            "description": "Build sinsp in minimal build with its tests",
-            "generator": "Unix Makefiles",
-            "binaryDir": "${sourceDir}/build-sinsp-minimal",
-            "cacheVariables": {
-                "CMAKE_BUILD_TYPE": "Release",
-                "BUILD_DRIVER": "ON",
-                "BUILD_BPF": "ON",
-                "USE_BUNDLED_DEPS": "ON",
-                "CREATE_TEST_TARGETS": "ON",
-                "MINIMAL_BUILD": "ON",
-                "SCAP_FILES_SUITE_ENABLE": "OFF"
-            }
-        }
-    ]
+	"version": 3,
+	"configurePresets": [
+		{
+			"name": "scap-drivers",
+			"displayName": "Build scap drivers and their tests",
+			"description": "Build all scap drivers (modern eBPF, legacy eBPF, kmod) and their tests",
+			"generator": "Unix Makefiles",
+			"binaryDir": "${sourceDir}/build-scap-drivers",
+			"cacheVariables": {
+				"CMAKE_BUILD_TYPE": "Release",
+				"BUILD_BPF": "ON",
+				"BUILD_DRIVER": "ON",
+				"USE_BUNDLED_DEPS": "ON",
+				"ENABLE_DRIVERS_TESTS": "ON",
+				"MODERN_BPF_DEBUG_MODE": "ON",
+				"BUILD_LIBSCAP_MODERN_BPF": "ON",
+				"BUILD_LIBSCAP_GVISOR": "OFF",
+				"CREATE_TEST_TARGETS": "ON",
+				"ENABLE_LIBSCAP_TESTS": "ON",
+				"SCAP_FILES_SUITE_ENABLE": "OFF",
+				"CMAKE_EXPORT_COMPILE_COMMANDS": "ON",
+				"BUILD_WARNINGS_AS_ERRORS": "ON"
+			}
+		},
+		{
+			"name": "sinsp-minimal",
+			"displayName": "Build sinsp in minimal build",
+			"description": "Build sinsp in minimal build with its tests",
+			"generator": "Unix Makefiles",
+			"binaryDir": "${sourceDir}/build-sinsp-minimal",
+			"cacheVariables": {
+				"CMAKE_BUILD_TYPE": "Release",
+				"BUILD_DRIVER": "ON",
+				"BUILD_BPF": "ON",
+				"USE_BUNDLED_DEPS": "ON",
+				"CREATE_TEST_TARGETS": "ON",
+				"MINIMAL_BUILD": "ON",
+				"SCAP_FILES_SUITE_ENABLE": "OFF",
+				"CMAKE_EXPORT_COMPILE_COMMANDS": "ON",
+				"BUILD_WARNINGS_AS_ERRORS": "ON"
+			}
+		}
```

Comment:
Not something for this PR, but we could create a `default` preset in order to reduce some of the configuration.
```suggestion
	"configurePresets": [
		{
			"name": "default",
			"generator": "Unix Makefiles",
			"cacheVariables": {
				"CMAKE_BUILD_TYPE": "Release",
				"BUILD_DRIVER": "ON",
				"BUILD_BPF": "ON",
				"USE_BUNDLED_DEPS": "ON",
				"CREATE_TEST_TARGETS": "ON",
				"MINIMAL_BUILD": "ON",
				"SCAP_FILES_SUITE_ENABLE": "OFF",
				"CMAKE_EXPORT_COMPILE_COMMANDS": "ON",
				"BUILD_WARNINGS_AS_ERRORS": "ON"
			}
		},
		{
			"name": "scap-drivers",
			"displayName": "Build scap drivers and their tests",
			"description": "Build all scap drivers (modern eBPF, legacy eBPF, kmod) and their tests",
			"inherits": "default",
			"binaryDir": "${sourceDir}/build-scap-drivers",
			"cacheVariables": {
				"CMAKE_BUILD_TYPE": "Release",
				"ENABLE_DRIVERS_TESTS": "ON",
				"MODERN_BPF_DEBUG_MODE": "ON",
				"BUILD_LIBSCAP_MODERN_BPF": "ON",
				"BUILD_LIBSCAP_GVISOR": "OFF",
				"ENABLE_LIBSCAP_TESTS": "ON"
			}
		},
		{
			"name": "sinsp-minimal",
			"displayName": "Build sinsp in minimal build",
			"description": "Build sinsp in minimal build with its tests",
			"inherits": "default",
			"binaryDir": "${sourceDir}/build-sinsp-minimal"
		}
```
You might need to also specify `binaryDir` for the default profile, I haven't tested the change.